CMakeLists.txt

```diff
@@ -19,6 +19,11 @@ option(SHERPA_ONNX_ENABLE_PORTAUDIO "Whether to build with portaudio" ON)
 option(SHERPA_ONNX_ENABLE_JNI "Whether to build JNI internface" OFF)
 option(SHERPA_ONNX_ENABLE_C_API "Whether to build C API" ON)
 option(SHERPA_ONNX_ENABLE_WEBSOCKET "Whether to build webscoket server/client" ON)
+option(SHERPA_ONNX_ENABLE_GPU "Enable ONNX Runtime GPU support" OFF)
+
+if(SHERPA_ONNX_ENABLE_GPU)
+  add_definitions(-DENABLE_GPU)
```

Comment:
Could you please move it to
https://github.com/k2-fsa/sherpa-onnx/blob/aa7108729bd07e773f03ea1a49eb8bb27367ff8b/sherpa-onnx/csrc/CMakeLists.txt#L76

`add_definitions` defines a macro in the global scope but we want to limit it only to `sherpa-onnx-core`.